src/js/components/RadioButton/StyledRadioButton.js

```diff
@@ -23,7 +23,16 @@ const StyledRadioButtonContainer = styled.label`
     border-color: ${props =>
       normalizeColor(props.theme.radioButton.hover.border.color, props.theme)};
   }
-
+  :hover {
+    background-color: ${props =>
+      normalizeColor(
+        !props.disabled &&
+          props.theme.radioButton.hover &&
+          props.theme.radioButton.hover.background &&
+          props.theme.radioButton.hover.background.color,
+        props.theme,
+      )};
+  }
```

Comment:
Why isn't this block of code part of line 23? why are we opening a new hover block instead of leveraging an existing one?

Reply:
Line 23 is adding a border color on hover to the child div that is contained inside.
we want the background color for hover to go on the parent div not the child. Therefore we need a new block of code.

Reply:
If you look at the 
https://deploy-preview-4327--sad-tereshkova-173d07.netlify.app/?path=/story/radiobuttongroup--custom-theme
the difference between child div getting the border color on hover and the background is. easier to see

Reply:
Thank you for the clarification.